django_freeradius/api/serializers.py

```diff
@@ -63,3 +65,53 @@ def validate(self, data):
     class Meta:
         model = RadiusAccounting
         fields = '__all__'
+
+
+class GroupSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Group
+        fields = '__all__'
+
+
+class UserSerializer(serializers.ModelSerializer):
+    groups = GroupSerializer(many=True)
+
+    class Meta:
+        model = User
+        fields = '__all__'
+
+
+class RadiusBatchCsvSerializer(serializers.ModelSerializer):
+    name = serializers.CharField()
```

Comment:
are you sure this is needed? Try to remove it, the `ModelSerializer` class should be able to determine this field automatically.
Same applies to all the other model fields for which you don't need to do customizations